src/blosc2/lazyexpr.py

```diff
@@ -86,6 +86,27 @@ def eval(self, item, **kwargs):
         * If self is a LazyArray from an udf, the kwargs used to store the resulting
           array will be the ones passed to the constructor in :func:`lazyudf` (except the
           `urlpath`) updated with the kwargs passed when calling this method.
+
+        Examples
+        --------
+        >>> import blosc2
+        >>> import numpy as np
+        >>> dtype = np.float64
+        >>> shape = [3, 3]
+        >>> size = shape[0] * shape[1]
+        >>> a = np.linspace(0, 5, num=size, dtype=dtype).reshape(shape)
+        >>> b = np.linspace(0, 5, num=size, dtype=dtype).reshape(shape)
+        >>> #  Convert numpy arrays to Blosc2 arrays
+        >>> a1 = blosc2.asarray(a)
+        >>> b1 = blosc2.asarray(b)
+        >>> # Perform the mathematical operation
+        >>> expr = a1 + b1
+        >>> output = expr.eval()
+        >>> f"Result of a + b (Lazy evaluation): {output[:]}"
+        Result of a + b (Lazy evaluation):
```

Comment:
Please do not use upper case in the middle of sentences.  For example, above "Lazy" -> "lazy".